### DISCUSSION TITLE
WMV after cut, odd Framerate values reported

### DISCUSSION THREAD

**Discussion by r3538987** [mifi/lossless-cut#1588] [Q&A]
Hello, what could be the reason, that once WMV is being cut via Lossless-Cut in certain systems reported framerate can be screwed up?
Below are some samples, original file, and two cut files, one processed with LosslessCut and other one with ASFbin 1.8.2.  

![WMV_after_LosslessCut](https://user-images.githubusercontent.com/33725345/235161361-08d20c86-514a-4de4-ba10-6d1eea73e024.jpg)

_InfoTips and Framerate column being shown is from app DirectoryOpus. In example if I enable such column in default explorer, it simply does not report anything for LC file._ 

Could the reason be that **Description of the codec** gets changed a bit, and this affects further mediainfo reading? 
Video and audio IDs are also swapped, but don't think that's would affect thing.

**Comment by mifi** [comment]
Hi. This is probably due to a bug in ffmpeg. Not sure how to fix it, maybe you can try to mux the video into a different containe rformat (matroska might work?) See related issue #1027

**Comment by r3538987** [comment]
When WMV video is in MKV container, on my side apps rarely detect metadata correctly via libs. Yes MediaInfo will show info still, but not libs. 
So once WMV pushed into MKV, explorer apps doesn't show fps values at all. 
Not a huge deal, but though there is info about it, because this was observed for years.

**Reply by mifi** [reply]
Ok, unfortunately I don't know how to fix this.
